automatic/vscode-insiders.install/vscode-insiders.install.nuspec

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>vscode-insiders.install</id>
+    <title>Visual Studio Code Insiders (Install)</title>
+    <version>1.46.0-insider</version>
```

Comment:
Just a note, remember that this version needs to be lower than the currently available insider version to make the updater work.